plugins/it.xsemantics.dsl/src/it/xsemantics/dsl/XsemanticsModelGenerator.xtend

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
```

Comment:
where does this come from? Is it brand new? comment headers have already been handled by Xsemantics for a long time now... @JensN4 added this feature some time ago...